pyproject.toml

```diff
@@ -26,10 +26,6 @@ no_implicit_optional = false  # TODO: PEP484, set to true
 strict = false  # TODO: set to true
 exclude = ["tests"]
 
-[[tool.mypy.overrides]]
```

Comment:
This is the most relevant line, the one we are trying to delete to make mypy more comprehensive.